lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -37,6 +37,10 @@ def descriptions(version)
       end
     end
 
+    def all_questions
+      dataset.keys.inject({}){|versions, version| versions.merge(questions(version))}
```

Comment:
I disagree. Using questions (or all_questions) would not make it clear that we are only interested in the questions that apply that version (current or future) of the calculator and not every single question that is defined in benefit-cap-calculator.